design-documents/catalog-domain/understanding-of-complex-product-types.md

```diff
@@ -0,0 +1,33 @@
+# Understanding of complex product types
+
+## 1. Definitions
+See [Simple Products](simple-products.md)
+* Complex product is a virtual representation of multiple real SKUs at the storefront.
+* Currently, Magento does support three complex product types - Bundle, Grouped and Configurable.
+* For the simplicity and data consistency a complex product has SKU.
+In fact this SKU is virtual.
+* Complex product may have all attributes that required to render product details page. 
```

Comment:
"may" means that it may also not have them. But later in the sentence we have "required to render product details page". How is it possible then to render the PDP if required attributes are absent? Though the idea makes sense, maybe it needs to be rephrased somehow.